.github/workflows/kv-vault.yml

```diff
@@ -118,7 +118,7 @@ jobs:
 
       - name: Determine artifact name
         run: |
-          echo "artifact-name=$(cargo metadata --no-deps --format-version 1 | jq -r '.packages[0].targets[0].name')" >> $GITHUB_ENV
+          echo "artifact-name=$(cargo metadata --no-deps --format-version 1 | jq -r '.packages[0].targets[1].name')" >> $GITHUB_ENV
```

Comment:
What's this?

Reply:
It makes it so that the script uses the binary output name instead of the library output name